var/spack/repos/builtin/packages/py-gosam/package.py

```diff
@@ -11,17 +11,20 @@ class PyGosam(PythonPackage):
        one-loop amplitudes for multi-particle processes in renormalizable
        quantum field theories."""
 
-    homepage = "https://gosam.hepforge.org"
-    url      = "https://gosam.hepforge.org/downloads/?f=gosam-2.0.4-6d9f1cba.tar.gz"
+    homepage = "https://github.com/gudrunhe/gosam"
+    git      = "https://github.com/gudrunhe/gosam.git"
 
     tags = ['hep']
 
-    version('2.0.4', sha256='faf621c70f66d9dffc16ac5cce66258067f39f686d722a4867eeb759fcde4f44')
+    version('2.1.1', tag='2.1.1')
+    version('2.0.4', sha256='faf621c70f66d9dffc16ac5cce66258067f39f686d722a4867eeb759fcde4f44',
+            url='https://gosam.hepforge.org/downloads/?f=gosam-2.0.4-6d9f1cba.tar.gz')
 
     depends_on('form', type='run')
     depends_on('qgraf', type='run')
     depends_on('gosam-contrib', type='link')
-    depends_on('python@2.7:2.7.99', type=('build', 'run'))
+    depends_on('python@2.7:2.7.99', type=('build', 'run'), when='@2.0.4')
```

Comment:
Maybe `@:2.0.4` in case anyone adds 2.0.3 (which is tagged in the git repo)?

Reply:
Will do.